pkg/scheduler/actions/reclaim/reclaim.go

```diff
@@ -49,6 +49,8 @@ func (ra *Action) Execute(ssn *framework.Session) {
 	klog.V(3).Infof("There are <%d> Jobs and <%d> Queues in total for scheduling.",
 		len(ssn.Jobs), len(ssn.Queues))
 
+	clearStarvedJobInOverusedQueue(ssn)
```

Comment:
reclaim the job in overused queue should be implemented in proportion plugin and called by reclaim action.